support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -315,6 +315,10 @@ export function ThreeTierLanding({
 		label: 'testing',
 		countdownStartInMillis: Date.parse('01 Jan 1970 00:00:00 GMT'),
 		countdownDeadlineInMillis: Date.parse('01 Jan 1970 00:00:00 GMT'),
+		theme: {
```

Comment:
why do we define some settings here? It can just come from `campaignSettings`

Reply:
It's setting the initial state of the countdown campaign so that it is not visible - I've moved it into campaign.tsx and imported it to reduce the size of the threetierlanding.

Reply:
I don't think initial config like this should be necessary. If a countdown should be displayed then we can use `campaignSettings?.countdownSettings`